recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - python
```

Comment:
Can you add the Python pin here again and rename `build` to `host`? This has been changed 4y ago but the recipe hasn't been touched since then.